_sass/custom/_components/buttons.scss

```diff
@@ -0,0 +1,190 @@
+// Buttons
+// ==========================
+
+.usa-button,
+.usa-button:visited,
+button,
+[type='button'],
+[type='submit'],
+[type='reset'],
+[type='image'] {
+  background-color: $color-medium;
+  font-family: $font-sans;
+  width: auto;
+
+  &:hover {
+    background-color: $color-medium-hover;
+  }
+
+  &.usa-button-secondary,
+  &.usa-button-secondary:visited {
+    background-color: $color-bright;
+    color: color('black');
+
+    &:hover {
+      background-color: $color-bright-hover;
+    }
```

Comment:
For some reason, the `color: color('black');` is getting overridden so that on `:hover` the DOM's computed color is the same as the `background-color`. See screenshot. 

![Screen Shot 2021-01-04 at 9 13 50 AM](https://user-images.githubusercontent.com/1093423/103556281-c695b080-4e6e-11eb-85bb-ecdbc9a33ee2.png)

```suggestion
      color: color('black');
    }
```

Perhaps adding the `color: color('black');` within the `&:hover` section will be a quick fix for this issue?

Reply:
OK... I tested this locally and it does fix the issue. I don't know if it's the best way to fix it though, as it *should* not need to be repeated using SASS, right?

Reply:
Ah, good catch. I think what we're looking for is the default ["accent cool" button](https://designsystem.digital.gov/components/button/) behavior, so we may be able to fix that by removing extra styling for that button.

Reply:
There was a similar issue with `:focus` that I [tried to override for that button](https://github.com/18F/portfolios/blob/01aca86b79e4f18aa86cdc176dc6d624c5b34aa5/_sass/custom/_overrides/portfolio-page.scss#L5), but taking that out reintroduces the issue from before (invisible text on `:focus`). We may need to unravel a bit further to get back to USWDS default, which is fine by me.

Reply:
Okay, check it out now.

Reply:
🥇  :shipit: